workflow/rules/annotation/pass1.smk

```diff
@@ -1,14 +1,29 @@
+rule download_snpeff_db:
+    output:
+        directory("results/refs/snpeff/{ref}")
```

Comment:
Not sure if you have decided on something else here, but generally the idea was to put references and databases in the main directory `resources` to distinguish it from input-data-specific `results`.

The general setup considerations are documented in this fork of the snakemake-workflow cookie-cutter template:
https://github.com/dlaehnemann/cookiecutter-snakemake-workflow/tree/cleaner-main-directory/%7B%7B%20cookiecutter.repo_name%20%7D%7D

And some more discussion is available in the respective PR:
https://github.com/snakemake-workflows/cookiecutter-snakemake-workflow/pull/4

Reply:
That is actually a good point. I agree. Let us fix this in a separate follow-up PR. I'll take care of that. Sorry for having forgotten that.